crates/oxc_transformer/Cargo.toml

```diff
@@ -30,13 +30,15 @@ oxc_span = { workspace = true }
 oxc_syntax = { workspace = true, features = ["to_js_string"] }
 oxc_traverse = { workspace = true }
 
+base64 = { workspace = true }
 dashmap = { workspace = true }
 indexmap = { workspace = true }
 oxc-browserslist = { workspace = true }
 ropey = { workspace = true }
 rustc-hash = { workspace = true }
 serde = { workspace = true, features = ["derive"] }
 serde_json = { workspace = true }
+sha1 = { version = "0.10.6" }
```

Comment:
Seems like a really expensive dependency :-/

Reply:
We use the same crate as swc
https://github.com/swc-project/swc/blob/12488da4b2c9fd150352e961f316801eeffa39a1/crates/swc_ecma_transforms_react/src/refresh/hook.rs#L106-L108